model/browser/events.yaml

```diff
@@ -52,3 +52,92 @@ groups:
           brief: A unique ID representing this particular metric instance.
           requirement_level: required
           examples: ["v3-1677874579383-6381583661209"]
+  - id: event.browser.user_action
+    stability: development
+    type: event
+    name: browser.user_action
+    brief: >
+      This event describes actions performed by the user such as click, scroll, zoom, resize, etc.
+    body:
+      id: browser.user_action
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: element
```

Comment:
I wonder if we should include "name" here to be clear what this field represents (the name of the element). And if we do that, should we just be consistent with the API and call it "tagName" instead?